yoti_python_sdk/__init__.py

```diff
@@ -22,10 +23,14 @@
 
 __version__ = main_ns["__version__"]
 YOTI_API_URL = environ.get("YOTI_API_URL", DEFAULTS["YOTI_API_URL"])
-YOTI_API_PORT = environ.get("YOTI_API_PORT", DEFAULTS["YOTI_API_PORT"])
+YOTI_API_PORT = environ.get('YOTI_API_PORT', DEFAULTS["YOTI_API_PORT"])
 YOTI_API_VERSION = environ.get("YOTI_API_VERSION", DEFAULTS["YOTI_API_VERSION"])
-YOTI_API_ENDPOINT = "{0}:{1}/api/{2}".format(
-    YOTI_API_URL, YOTI_API_PORT, YOTI_API_VERSION
-)
+YOTI_API_ENDPOINT = environ.get("YOTI_API_ENDPOINT", "{0}:{1}/api/{2}".format(YOTI_API_URL, YOTI_API_PORT, YOTI_API_VERSION))
+
+YOTI_API_VERIFY_SSL = environ.get("YOTI_API_VERIFY_SSL", DEFAULTS["YOTI_API_VERIFY_SSL"])
+if YOTI_API_VERIFY_SSL == "false" or YOTI_API_VERIFY_SSL == False:
```

Comment:
Although it might be excessive to write a helper function here, I think we need to at least also check for `"False"` if we are to check for `"False"` and `False`